### DISCUSSION TITLE
빌드 및 배포 에러

### DISCUSSION THREAD

**Discussion by DoItAmelia** [devrel-kr/HackaLearn#62] [2 Q&A - 클라우드 스킬 챌린지] [answered]
안녕하세요!
클라우드 챌린지
> Gatsby 및 Azure Static Web Apps를 사용하여 정적 웹앱 만들기 및 게시 
> 연습 - Azure Static Web Apps 빌드 및 배포: https://docs.microsoft.com/ko-kr/learn/modules/create-deploy-static-webapp-gatsby-app-service/7-exercise-build-deploy-static-app
> GitHub 작업 검토
: 아래 에러와 함께 빌드 및 배포작업이 제대로 이루어지지 않고 있습니다. 어떻게 하면 해당 에러를 해결할 수 있을까요?
에러에서 언급하고 있는 default file은 앞서 과정에서 언급한대로 이미 생성되어있습니다. 
감사합니다. 
![image](https://user-images.githubusercontent.com/88205708/128440833-fbaa3427-bae1-47b4-8a16-06b09e419dfd.png)
![image](https://user-images.githubusercontent.com/88205708/128441144-857eabcf-c473-447d-b23a-c385b34910c5.png)

**Comment by justinyoo** [comment, marked as answer]
최초 배포시 배포 경로를 인식하지 못하는 것으로 보입니다. 혹시 `.github/workflows/azure-static-web-apps-xxxx-xxxx-xxxx.yml` 이라는 파일 내용을 공유해 주실 수 있으세요?

**Reply by DoItAmelia** [reply]
안녕하세요. 파일 내용은 아래와 같습니다. 
![image](https://user-images.githubusercontent.com/88205708/128443818-5d60d131-276e-4e45-b4bf-45a719af63ef.png)

**Reply by justinyoo** [reply]
리포지토리가 [이거](https://github.com/DoItAmelia/gatsby-app) 맞죠? 지금 깃헙 액션 설정에 보면

`app_location` 값이 `/`로 되어 있는데, 현재 리포지토리 구조를 보면 이 값을 `myApp`으로 바꿔 보면 될 것 같아요.

**Reply by DoItAmelia** [reply]
감사합니다. 
Azure Portal에서 Static Web Apps을 만들때 App위치를 기본값이였던 / 대신 myApp을 입력하여 해결하였습니다. 
도움 주셔서 다시 한번 감사드립니다:)

1) 수정 위치
![image](https://user-images.githubusercontent.com/88205708/128447848-1ccc0b6a-cfdb-4c56-a613-7c3d34c716d7.png)

2) Build and Deploy 결과
![image](https://user-images.githubusercontent.com/88205708/128447891-4db3cc9a-2bf0-4cd3-a722-e7a27908d38d.png)

3) 만들어진 웹 화면
![image](https://user-images.githubusercontent.com/88205708/128447797-ad5d96fb-926c-4fd5-b201-43037a3e7050.png)

**Reply by justinyoo** [reply]
@DoItAmelia  답으로 채택해 주시면 비슷한 상황을 겪는 다른 분들도 쉽게 찾으실 수 있을 거예욥!
